docs/sdk/authentication/session-sigs/get-session-sigs.md

```diff
@@ -3,112 +3,256 @@ sidebar_position: 2
+  
+If you want to request the ability to decrypt an access control condition, then you would pass:
+   ```ts
+   [{ resource: new LitAccessControlConditionResource('someResource'), ability: LitAbility.AccessControlConditionDecryption }]
+   ```
+  
+### `sessionCapabilityObjects`
+  The session capability object that you want to request for this session. If you pass nothing, then this will default to
+  wildcard for each type of resource you're accessing.
+  For example, if you passed nothing, and you're requesting to perform a decryption operation for an access control condition, then the session capability object will be a wildcard for the access control condition, which grants this session signature the ability to decrypt this access control condition.
+
+### `switchChain`
+If you want to ask MetaMask to try and switch the user's chain, you may pass true here.  This will only work if the user is using MetaMask.  If the user is not using MetaMask, then this will be ignored.
+
+### `sessionKey`
+The serialized session key pair to sign. If not provided, a session key pair will be fetched from localStorage or generated.
+
+### `capabilityAuthSigs`
+Not limited to capacityDelegationAuthSig, we want to be able to pass in any other authSigs for other purposes.
+
+### `authNededCallback`
+This is a callback that will be called if you need to authenticate using a PKP. If you don't pass this callback, then the user will be prompted to authenticate with their wallet, like MetaMask.
+
+## Return Value
+
+### `sig`
+The signature produced by the ed25519 key pair signing the `signedMessage` payload.
+
+### `derivedVia`
+Should be `litSessionSignViaNacl`, specifies that the session signature object was created via the `NaCl` library.
+
+### `signedMessage`
+The payload signed by the session key pair. This is the signed `AuthSig`, which includes the fields:
+
+- #### `sessionKey`
+   The session key pair public key.
+- #### `resourceAbilityRequests`
+   An array of resource abilities that you have requested for the session.
+- #### `capabilities` <- Needs a fact check
+   An array of one or more `AuthSig`. The `capabilities` authorize the `AuthSig` address(es) to utilize the resources specified. in the `resourceAbilityRequests`
+- #### `issuedAt`
+   The time the session signature was issued.
+- #### `expiration`
+   The time the session signature becomes invalid.
+- #### `nodeAddress`
+   The specific URL the session signature is meant for.
+
+### `address`
+The session key pair public key.
+
+### `algo`
+The signing algorithm used to generate the session signature.
+
+## Example Implementation
+
+Now that we know what the `getSessionSigs` function does, it's parameters, and it's return values, let's dig into a complete implementation.
+
+The full code implementation can be found [here](https://github.com/LIT-Protocol/developer-guides-code/tree/master/session-signatures/getSessionSigs). 
+
+### Installing the Required Dependencies
```

Comment:
We should add a note to this section that the `node-localstorage` dependency is only required when executing outside of a browser environment since the SDK will use the native browser storage